### ISSUE TITLE
Training issues:low success rate and fast memory  increase

### ISSUE THREAD

**Issue by liliang18273158269** [Zeying-Gong/Falcon#8]
1、As the training log shows, the success rate(0.006) is very low on the training process even I have trained for 3020 steps and gotten the 10 checkpoints, below are some training logs :

2025-05-05 15:55:19,367 update: 3020	fps: 24.233	
2025-05-05 15:55:19,369 Num updates: 3020	Num frames 6279948
2025-05-05 15:55:19,370 Average window size: 50  did_multi_agents_collide: 0.195  distance_to_goal: 7.770  distance_to_goal_reward: 0.002  human_collision: 0.195  multi_agent_nav_reward: -0.001  num_steps: 414.275  
psc: 0.926  reward: -2.075  spl: 0.006  stl: 0.006  success: 0.006
2025-05-05 15:55:19,370 	Perf Stats: trainer.rollout_collect: 64.566 trainer.sample_action: 0.019 trainer.obs_insert: 0.001 
trainer.step_env: 0.504 trainer.update_stats: 0.003 trainer.update_agent: 24.190
PPO save to ckpt.10.pth

2、I have noticed that the memory usage increased very fast during the training process, from 150G to 250G within a day, which lead to my training's stop several times for the memory is full.

Could you please explain the reasons ,thank you very much.

**Comment by gkjgkd** [comment]
Is the  HM3D pretrained weights point to pretrained_habitat3.pth? how can I to check the HM3D pretrained weights are properly loaded before starting fine-tuning
